st2common/st2common/models/system/action.py

```diff
@@ -30,6 +30,7 @@
 from st2common.constants.action import LIBS_DIR as ACTION_LIBS_DIR
 from st2common.util.secrets import get_secret_parameters
 from st2common.util.secrets import mask_secret_parameters
+from st2common.logging.formatters import MASKED_ATTRIBUTE_VALUE
```

Comment:
Shouldn't this be from https://github.com/StackStorm/st2/blob/master/st2common/st2common/constants/secrets.py? Seems to me that we should only have `MASKED_ATTRIBUTE_VALUE` in one place.

Reply:
Yeah, it looks like we actually have the same constant in two places - we should try to consolidate that.